frontend/src/components/DownloadAnalysisLink/DownloadAnalysisLink.tsx

```diff
@@ -15,6 +16,9 @@ import { getLegend } from '../MapLegend/MapLegend'
 
 import './DownloadAnalysisLink.css'
 
+const useStaticCsvs: boolean = process.env.ENV === 'PRD'
```

Comment:
Could also use the config, or even add useStaticCsvs to the shared config

Reply:
Yeah let me put it in the config.

Reply:
pinging me on encompass reviews on a wednesday eh 📆